### DISCUSSION TITLE
Just discovered for the first time, that you can TRANSLATE from EN to another langage direclty while WHISPERING.

### DISCUSSION THREAD

**Discussion by NormalFall0** [openai/whisper#2176] [General]
So I selected FR by mistake, for an ENG video, and I noticed the results were in french.
And they seem to "fit" the real words pronounced in english.
I was expecting hallucination or an error.

**Comment by glangford** [comment]
You won't get this behaviour reliably. See

* https://github.com/openai/whisper/discussions/649
